### DISCUSSION TITLE
Seeking Guidance on Optimizing tDCS e-Field Strength

### DISCUSSION THREAD

**Discussion by ryokisasaki** [simnibs/simnibs#371] [8. Simulation workflows]
Dear simnibs community,

Hello everyone! I am currently trying to optimize the tDCS e-field strength using four electrodes. Please refere to the script code below. The goal of this optimization is to determine the optimal electrode positions and intensities to achieve a 0.20 V/m electric field. When I run this script, I obtain some optimized information, such as electrode positions and tDCS intensities. However, is it possible to use different intensities between four electrodes? If not, should I average them? I would appreciate any guidance you can provide.

--------------------------------------------------------------
%% tDCS_opt script

opt = opt_struct('TDCSoptimize');
% Subject folder
opt.subpath = s.subpath;

% Select output folder
opt.pathfem = s.subpath;

% Select a name for the optimization
opt.name = 'optimization/strength';

% Optimize controlling electric field strength
opt = opt_struct('TDCSoptimize');

% Select the leadfield file
opt.leadfield_hdf = char(['simnibs_leadfield\enrie_leadfield_EEG10-10_UI_Jurak_2007.hdf5']);

opt.max_total_current = 4e-3;
opt.max_individual_current = 1e-3;
opt.max_active_electrodes = 4;

% Target in the A region
opt.target(1).positions = A_MNI;
opt.target(1).intensity = 0.2;
opt.target(1).directions='none';

% Target in the B region
opt.target(2).positions  = B_MNI;
opt.target(2).intensity = 0.2;
opt.target(2).directions='none';

run_simnibs(opt);
------------------------------------------

Best regards,
Ryoki

**Comment by bjornsig** [comment]
Hi Ryoki,
Yes, that is what the optimization does. It determines the best intensities across the electrodes. We are not sure what you mean by average them, can you explain more?
Best regards,
Bjorn

**Reply by ryokisasaki** [reply]
> Hi Ryoki, Yes, that is what the optimization does. It determines the best intensities across the electrodes. We are not sure what you mean by average them, can you explain more? Best regards, Bjorn

Hi Bjorn,

Thank you so much for your guidance. Setting four different intensities is generally not allowed in typical tES devices. Therefore, I thought that these intensities should be averaged across channels. Please see the example below:

Optimized four intensities: 0.33, -0.43, 0.56, -0.45
Averaged four intensities: 0.44, -0.44, 0.44, -0.44

The optimized intensities were averaged using absolute values, with the second and fourth values made negative. I would appreciate it if you could provide further guidance.

Best regards,
Ryoki

**Reply by diedrim** [reply]
You could set up a normal tDCS simulation where you use the selected electrode positions and your averaged four intensities to see how close this solution is to the originally optimized solution. By that you at least know how good your stimulation will be.

**Reply by ryokisasaki** [reply]
Thank you for the suggestion. I will follow your advise and see how it goes.

Best regards,
Ryoki
